name = 'Samuel'
# raise Exception('Stop ada kesalahan') #tempat menghentikan program dimana saja, dan ngasih output sesuai yg kita mau

try:
    print(i + name)
except:
    print('Ada yang salah dalam i + name')